src/main/kotlin/apply/application/EvaluationTargetCsvService.kt

```diff
@@ -57,14 +59,17 @@ class EvaluationTargetCsvService(
                     .withTrim()
             )
             for (csvRecord in csvParser) {
-                var targetId = csvRecord.get(ID)
-                var name = csvRecord.get(NAME)
-                var email = csvRecord.get(EMAIL)
+                var evaluationTargetId = csvRecord.get(ID)
                 var evaluationStatus = csvRecord.getEvaluationStatus()
                 val evaluationAnswers = csvRecord.getEvaluationAnswers(evaluationItems)
-                // TODO: 평가 대상자 상태 업데이트 기능 구현
+                val note = csvRecord.get(NOTE)
+
+                val evaluationScores = evaluationAnswers.map { EvaluationItemScoreData(it.score, it.evaluationItemId) }
+                val evaluationTargetData = EvaluationTargetData(evaluationScores, note, evaluationStatus)
+                evaluationTargetsData[evaluationTargetId.toLong()] = evaluationTargetData
             }
         }
+        evaluationTargetService.updateGrades(evaluationId, evaluationTargetsData)
     }
 
     private fun CSVRecord.getEvaluationStatus(): EvaluationStatus = EvaluationStatus.valueOf(get(STATUS))
```

Comment:
```suggestion
    private fun CSVRecord.getEvaluationStatus(): EvaluationStatus = EvaluationStatus.valueOf(get(STATUS).uppercase())
```
여긴 사실 제가 해놓고 다시 생각이 들어서 남기는 부분인데요 ㅎㅎㅎㄹ
평가자가 평가 상태를 입력할 때 현재는 무조건 대문자로만 입력해야하는데,
대소문자 상관 없이 평가 상태만 맞으면 해당 상태를 찾아주면 작성하는 입장에서 좀 더 편리하지 않을까 하는데 어떨까요 ??

Reply:
좋은의견입니다~ 반영할게요!ㅎㅎㅎ